2022-04-17 15:25:42,233 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime + H2LeakageVol]))]
TypeError: must be str, not int
2022-04-17 15:25:42,825 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime + H2LeakageVol]))]
TypeError: must be str, not int
2022-04-17 15:25:43,393 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime + H2LeakageVol]))]
TypeError: must be str, not int
2022-04-17 15:25:43,943 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime + H2LeakageVol]))]
TypeError: must be str, not int
2022-04-17 15:25:44,511 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime + H2LeakageVol]))]
TypeError: must be str, not int
2022-04-17 15:25:45,097 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime + H2LeakageVol]))]
TypeError: must be str, not int
2022-04-17 15:26:15,973 INFO: LaiZhou startup [in C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\__init__.py:34]
2022-04-17 15:27:53,883 INFO: LaiZhou startup [in C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\__init__.py:34]
2022-04-17 15:29:01,075 INFO: LaiZhou startup [in C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\__init__.py:34]
2022-04-17 15:29:04,198 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 257, in HOW_pred
    H2FillJudge()
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 126, in H2FillJudge
    print(Config.x)
AttributeError: type object 'Config' has no attribute 'x'
